postgres_docker.sh

```diff
@@ -0,0 +1 @@
+docker run --name postgres-swlcgdb -e POSTGRES_DB=swlcgdb_development -e POSTGRES_USER=swlcgdb -e POSTGRES_PASSWORD=swlcgdb_development -p 5432:5432 -d postgres:16
```

Comment:
You need a carriage return (or linefeed, whatever you want to call it)  at the end of this line. It's a POSIX requirement.

https://stackoverflow.com/a/729795/5153779

Reply:
+1